Makefile

```diff
@@ -447,50 +412,70 @@ endif
 
 ## -- Bundle validation, push and release targets
 
-.PHONY: merge-to-master-release
+.PHONY: build-operator-image
 ## Make a dev release on every merge to master
-merge-to-master-release:
-	echo "${QUAY_TOKEN}" | $(CONTAINER_RUNTIME) login -u "redhat-developer+travis" --password-stdin quay.io
-	$(eval COMMIT_COUNT := $(shell git rev-list --count HEAD))
+build-operator-image:
+	@echo "${QUAY_TOKEN}" | $(CONTAINER_RUNTIME) login -u "$(QUAY_USERNAME)" --password-stdin quay.io
 	$(Q)$(CONTAINER_RUNTIME) build -f Dockerfile.rhel -t $(OPERATOR_IMAGE_REF) .
-	$(CONTAINER_RUNTIME) push "$(OPERATOR_IMAGE_REF)"
 
+.PHONY: push-operator-image
+## push operator image to quay
+push-operator-image: build-operator-image
+	$(CONTAINER_RUNTIME) push "$(OPERATOR_IMAGE_REF)"
 
-.PHONY: push-to-manifest-repo
-## Push manifest bundle to service-binding-operator-manifest repo
-push-to-manifest-repo:
-	@rm -rf $(MANIFESTS_TMP) || true
-	@mkdir -p ${MANIFESTS_TMP}/${BUNDLE_VERSION}
-	operator-sdk generate csv --csv-version $(BUNDLE_VERSION) --from-version=0.0.23
-	cp -vrf $(OLM_CATALOG_DIR)/$(GO_PACKAGE_REPO_NAME)/$(BUNDLE_VERSION)/* $(MANIFESTS_TMP)/$(BUNDLE_VERSION)/
-	cp -vrf $(OLM_CATALOG_DIR)/$(GO_PACKAGE_REPO_NAME)/*package.yaml $(MANIFESTS_TMP)/
-	cp -vrf $(CRDS_DIR)/*_crd.yaml $(MANIFESTS_TMP)/${BUNDLE_VERSION}/
-	sed -i -e 's,REPLACE_IMAGE,$(OPERATOR_IMAGE_REF),g' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
-	sed -i -e 's,CSV_CREATION_TIMESTAMP,$(CSV_CREATION_TIMESTAMP),g' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
-	awk -i inplace '!/^[[:space:]]+replaces:[[:space:]]+[[:graph:]]+/ { print $0 }' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
-	sed -i -e 's,BUNDLE_VERSION,$(BUNDLE_VERSION),g' $(MANIFESTS_TMP)/*.yaml
-	sed -i -e 's,PACKAGE_NAME,$(CSV_PACKAGE_NAME),g' $(MANIFESTS_TMP)/*.yaml
-	sed -i -e 's,ICON_BASE64_DATA,$(shell base64 --wrap=0 ./assets/icon/red-hat-logo.svg),g' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
-	sed -i -e 's,ICON_MEDIA_TYPE,image/svg+xml,g' $(MANIFESTS_TMP)/${BUNDLE_VERSION}/*.clusterserviceversion.yaml
-
-.PHONY: prepare-bundle-to-quay
-## Prepare manifest bundle to quay application
-prepare-bundle-to-quay:
+.PHONY: prepare-operator-manifests
+## Prepare operator manifests
+prepare-operator-manifests:
```

Comment:
this depends on `build-operator-image` as well

Reply:
This rather depends on `push-operator-image` because the image digest used in operator's deployment is available only after the image is pushed to the repo.